3 - PB/Documentazione interna/Verbali/24-03-30/24-03-30.typ

```diff
@@ -0,0 +1,104 @@
+#import "/template.typ": *
+
+#show: project.with(
+  date: "30/03/24",
+  subTitle: "Meeting di retrospettiva e pianificazione",
+  docType: "verbale",
+  authors: (
+    "Oseliero Antonio",
+  ),
+  reviewers: (
+    "Banzato Alessio",
+  ),
+  missingMembers: (
+    "Todesco Mattia",
+  ),
+  timeStart: "10:30",
+  timeEnd: "11:15",
+);
+
+= Ordine del giorno
+- Valutazione del progresso generale;
+- Pianificazione.
+- Invio candidatura PB;
+
+= Valutazione del progresso generale <avanzamento>
+
+Lo Sprint 21 è servito al gruppo per pianificare e continuare gli ultimi lavori in previsione del colloquio PB.
```

Comment:
- Riformulerei in "Durante lo Sprint 21 sono stati pianificati e continuati gli ultimi lavori in previsione del colloquio PB.";
- Aggiungere che è stato fatto un colloquio con il Proponente per la convalida del MVP.